packages/table/table.scss

```diff
@@ -0,0 +1,70 @@
+@import "~@fremtind/jkl-core/variables/_all.scss";
+
+$bottom-border-size: 1px;
+
+.jkl-table {
+    border-collapse: collapse;
+
+    &__anchor {
+        position: absolute;
+        width: 100%;
+        height: 100%;
+        opacity: 0;
+        top: 0;
+        left: 0;
+    }
+
+    &__heading {
+        color: $matt-svart;
+        font-size: $font-size-1;
+        font-weight: normal;
+        line-height: $line-height-1;
+        padding: $component-spacing--small 0.5rem;
+    }
+
+    &__data-cell {
+        position: relative;
+        box-sizing: border-box;
+        padding: 1.5rem 0.5rem;
+        line-height: $line-height-2;
+    }
+
+    &__heading,
+    &__data-cell {
+        text-align: left;
+        vertical-align: top;
+
+        &:first-of-type {
+            padding-left: 1rem;
+        }
+
+        &:last-of-type {
+            padding-right: 1rem;
+        }
+    }
+
+    &__row {
+        box-shadow: 0 ($bottom-border-size * 2) 0px -1px rgba(0, 0, 0, 0.2);
+
+        &--anchor-row {
+            cursor: pointer;
+        }
+
+        &--data-row {
+            transition: background-color 150ms ease, box-shadow 150ms ease;
+            &:hover {
+                background-color: white;
+                box-shadow: 0 4px 3px -3px rgba(0, 0, 0, 0.2);
+            }
+            &:active {
+                box-shadow: 0 ($bottom-border-size * 2) 0px -1px rgba(0, 0, 0, 0.4);
+            }
+        }
+
+        /* Focus-within is not supported by IE/Edge */
```

Comment:
😥

Reply:
I found no way of using css only to trigger focus state. Only other way is to use js and onFocus/onBlur and add focus-classes on the row. Maybe we should add this? @piofinn

Reply:
Yeah, might as well do that. Some accessibility is better than none! There is [a polyfill](https://www.npmjs.com/package/focus-within-polyfill) for focus-within that we should probably look into before we end up rewriting it ourselves...

Reply:
I added documentation about the polyfill.